src/wallet/spend.h

```diff
@@ -6,6 +6,7 @@
 #define BITCOIN_WALLET_SPEND_H
 
 #include <consensus/amount.h>
+#include <policy/fees.h> // for FeeCalculation
```

Comment:
we don't add those comments, as they are impossible to maintain

Reply:
k, got it. Thanks.